<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NielsenVis</class>
 <widget class="QMainWindow" name="NielsenVis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nielson Visualization</string>
  </property>
  <widget class="QWidget" name="TopKek">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="Controls">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="Sort">
         <property name="title">
          <string>Sort Order</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="Name">
            <property name="text">
             <string>By name</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Int">
            <property name="text">
             <string>By Interim DDI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Plan">
            <property name="text">
             <string>By Planning DDI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Final">
            <property name="text">
             <string>By Final DDI</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="AddGraph">
         <property name="title">
          <string>Add Graph</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QFrame" name="Chooser">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>By</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="PrimKey">
               <item>
                <property name="text">
                 <string>Demographic</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Market</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="CatDDown">
            <item>
             <property name="text">
              <string>Needs .CSV files.</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Add">
            <property name="text">
             <string>Add Graph!</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Interim">
         <property name="text">
          <string>Add interim .csv data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Planning">
         <property name="text">
          <string>Add planning .csv data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FinalCsv">
         <property name="text">
          <string>Add final .csv data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="RenderArea" name="Render" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
   <slots>
    <slot>chartAdded()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
